test/deprecated/io/checkreadfreg.skipif

```diff
@@ -0,0 +1 @@
+REGEXP!=re2
```

Comment:
This needs to be `CHPL_REGEXP`.  As it is, the test doesn't run when `CHPL_REGEXP` is set to re2 (because REGEXP isn't set in my path and so is not re2), which is when it should run